pom.xml

```diff
@@ -131,6 +131,11 @@
             <dependency>
                 <groupId>net.minidev</groupId>
                 <artifactId>json-smart</artifactId>
+                <version>2.4.5</version>
```

Comment:
Why not to upgrade `json-smart` to 2.4.7 that is more widely used compared to 2.4.5?

Reply:
Additionally, it will be better to remove the dependency on json-smart completely.

Reply:
> Additionally, it will be better to remove the dependency on json-smart completely.
Tried replacing `json-smart` with [Jackson](https://github.com/opensearch-project/security/pull/1299/commits/667725ed14b2eee7a4e8e42573a36154f89485c7), but Jackson automatically wraps the values in Json with [quotes](https://github.com/FasterXML/jackson-databind/issues/715#issuecomment-76782907), which fails the tests currently relying on `json-smar`. The following is the failure:
```
org.junit.ComparisonFailure: 
Expected :{"http_certificates_list":"[{issuer_dn=CN=Example Com Inc. Signing CA,OU=Example Com Inc. Signing CA,O=Example Com Inc.,DC=example,DC=com, subject_dn=CN=node-1.example.com,OU=SSL,O=Test,L=Test,C=DE, san=[[2, node-1.example.com], [2, localhost], [7, 127.0.0.1], [8, 1.2.3.4.5.5]], not_before=2020-02-17T16:19:25Z, not_after=2022-02-16T16:19:25Z}]","transport_certificates_list":"[{issuer_dn=CN=Example Com Inc. Signing CA,OU=Example Com Inc. Signing CA,O=Example Com Inc.,DC=example,DC=com, subject_dn=CN=node-1.example.com,OU=SSL,O=Test,L=Test,C=DE, san=[[2, node-1.example.com], [2, localhost], [7, 127.0.0.1], [8, 1.2.3.4.5.5]], not_before=2020-02-17T16:19:25Z, not_after=2022-02-16T16:19:25Z}]"}
Actual   :{"http_certificates_list":[{"issuer_dn":"CN=Example Com Inc. Signing CA,OU=Example Com Inc. Signing CA,O=Example Com Inc.,DC=example,DC=com","subject_dn":"CN=node-1.example.com,OU=SSL,O=Test,L=Test,C=DE","san":"[[2, node-1.example.com], [2, localhost], [7, 127.0.0.1], [8, 1.2.3.4.5.5]]","not_before":"2020-02-17T16:19:25Z","not_after":"2022-02-16T16:19:25Z"}],"transport_certificates_list":[{"issuer_dn":"CN=Example Com Inc. Signing CA,OU=Example Com Inc. Signing CA,O=Example Com Inc.,DC=example,DC=com","subject_dn":"CN=node-1.example.com,OU=SSL,O=Test,L=Test,C=DE","san":"[[2, node-1.example.com], [2, localhost], [7, 127.0.0.1], [8, 1.2.3.4.5.5]]","not_before":"2020-02-17T16:19:25Z","not_after":"2022-02-16T16:19:25Z"}]}
```

Reply:
@cliu123 I don't see why jackson is not an option. The expected is not properly constructed.